docs/source/existing_code.rst

```diff
@@ -0,0 +1,174 @@
+
+* Specify mypy config file or command-line options.
+
+* Provide set of files to type check. You may want to implement
+  inclusion and exclusion filters for full control of the file
+  list.
+
+Continuous Integration
+----------------------
+
+Once you have a clean mypy run and a runner script for a part
+of your codebase, set up your Continuous Integration (CI) system to
+run mypy to ensure that developers won't introduce bad annotations.
+A simple CI script could look something like this:
+
+.. code-block:: text
+
+    python3 -m pip install mypy==0.600  # Pinned version avoids surprises
+    scripts/mypy  # Runs with the correct options
+
+Annotate library modules
+------------------------
+
+Most projects have some commonly used library code. It's a good idea
+to annotate these pretty early on, since this allows code using these
+modules to be type checked more effectively. Since mypy supports
+gradual typing, it's okay to leave some unannotated library
+modules. The more you annotate, the more useful mypy will be, but even
+a little annotation coverage is useful.
+
+Write annotations as you go
+---------------------------
+
+Now you are ready to include type annotations in your development
+workflows. Establish in your code style conventions that developers
+should add annotations for any new code, and it's also a good idea to
+write annotations when you modify existing code.  This way you'll
+gradually increase annotation coverage in your codebase without
+much effort.
+
+Automate annotation of legacy code
+----------------------------------
+
+There are tools for automatically adding draft annotations
+based on type profiles collected at runtime.  Tools include
+`MonkeyType <https://github.com/Instagram/MonkeyType>`_
+(Python 3) and `PyAnnotate <https://github.com/dropbox/pyannotate>`_
+(type comments only).
+
+A simple approach is to collect types from test runs. This may work
+well if your test coverage is good (and if your tests aren't very
+slow).
+
+Another approach is to enable type collection for a small, random
+fraction of production network requests.  This clearly requires more
+care, as type collection could impact the reliability or the
+performance of your service.
+
+Speed up mypy runs
+------------------
+
+You can use :ref:`mypy daemon <mypy_daemon>` to get much faster
+incremental mypy runs. The larger your project is, the more useful
+this will be.  If your project has at least 100,000 lines of code or
+so, you may also want to set up :ref:`remote caching <remote-cache>`
+for further speedups.
+
+Introduce stricter options
+--------------------------
+
+Mypy is very configurable. Once you get started with static typing,
+you may want to explore the various stricness options mypy provides to
+catch more bugs. For example, you can ask mypy to require annotations
+for all functions in certain modules to avoid accidentally introducing
+code that won't be type checked.
```

Comment:
It might be nice to link to the command line or config file sections somewhere in this paragraph.